src/AppInstallerRepositoryCore/Microsoft/PreIndexedPackageSourceFactory.cpp

```diff
@@ -216,8 +208,17 @@ namespace AppInstaller::Repository::Microsoft
 
             void RemoveInternal(const SourceDetails& details, IProgressCallback& callback) override
             {
-                AICLI_LOG(Repo, Info, << "Removing package: " << GetPackageFullNameFromDetails(details));
-                Deployment::RemovePackage(GetPackageFullNameFromDetails(details), callback);
+                auto fullName = Msix::GetPackageFullNameFromFamilyName(GetPackageFamilyNameFromDetails(details));
+
+                if (!fullName)
+                {
+                    AICLI_LOG(Repo, Info, << "No full name found for family name: " << GetPackageFamilyNameFromDetails(details));

```

Comment:
>No full name found for family name [](start = 46, length = 34)

Could it be the case that multiple found?

Reply:
Yes, it is possible, but that case is supposed to be a bug.  

---
In reply to: [444598040](https://github.com/Microsoft/pkg/pull/455#discussion_r444598040) [](ancestors = 444598040)